lte/gateway/python/magma/pipelined/tests/test_access_control.py

```diff
@@ -132,11 +105,73 @@ def setUpClass(cls):
             access_control_controller_reference.result()
         cls.testing_controller = testing_controller_reference.result()
 
+    @classmethod
+    def getConfig(cls):
+        raise NotImplementedError
+
+    @classmethod
+    def getMconfig(cls):
```

Comment:
Normally, function names should be snake case. (Unittest only has some as camel case because it started as a port of junit.)